core/custom_field_api.php

```diff
@@ -1433,6 +1433,15 @@ function print_custom_field_input( array $p_field_def, $p_bug_id = null, $p_requ
 	}
 }
 
+/**
+ * Constructs a string that can be used as part of a CSS class name based on a custom field name
+ * @param string $p_custom_field_name The custom field name
+ * @return string The CSS string
+ */
+function custom_field_css_name( $p_custom_field_name ) {
+    return 'custom-' . str_replace( ' ', '-', $p_custom_field_name );
```

Comment:
To be on the safe side, instead of just dealing with spaces, I would suggest using a regex to also take care of any sequence of potentially invalid chars, and replace them with a dash - something like (offhand, not tested)

```
preg_replace( '/[^A-Za-z0-9_-]+/', '-', $p_custom_field_name );
```